### PR TITLE
[ #13 ] - Character selection screen works

### PR DESCRIPTION
**Pull request by BrianDassow**

### CODE REVIEW

File: Game.gd [hunk 1]
```diff
@@ -4,28 +4,18 @@ func _ready():
 	get_tree().connect('network_peer_disconnected', self, '_on_player_disconnected')
 	get_tree().connect('server_disconnected', self, '_on_server_disconnected')
 	
-	var new_player = preload('res://characters/player/Player.tscn').instance()
-	var new_dungeon = preload('res://levels/dungeon/dungeon.tscn').instance()
+	createLevel()
+	PlayerHandler.createPlayer(str(get_tree().get_network_unique_id()), PlayerInformation.userType, (get_tree().get_network_unique_id()), true)
```
Comment (Ex3Dophos) [inline Game.gd]: For the 3rd parameter, I see that it is in brackets. Did you forget to cast to a string? Or is this just an unnecessary use of brackets?

Comment (BrianDassow) [reply Game.gd]: accident from copying and pasting. doesnt need the brackets

File: characters/player/Player.gd [hunk 2]
```diff
@@ -69,8 +72,8 @@ func _physics_process(delta):
 		_move(slave_movement)
 		position = slave_position
 
-	if get_tree().is_network_server():
-		Network.update_position(int(name), position)
+	#if get_tree().is_network_server():
+	#	Network.update_position(int(name), position)
```
Comment (Ex3Dophos) [inline characters/player/Player.gd]: Remove these two lines completely?

Comment (BrianDassow) [reply characters/player/Player.gd]: can, was going to let you refactor player

Comment (Ex3Dophos) [reply characters/player/Player.gd]: okay, that's fine. I can clean up things later then.

**Pull request merged**